### PR TITLE
remove duplicate RX/TX pin lines

### PR DESCRIPTION
**Pull request by jepler**
These are duplicated above, next to the D0/D1 version of the pins; check out the full file to see it, it's outside the normal diff context.

### CODE REVIEW

**Comment by dhalbert** [comment]
They are reversed in one pair versus the other. Which one is right? TX/RX is 10/9 or 9/10?

**Comment by jepler** [comment]
oh my, I'll double check!

**Comment by jepler** [comment]
```
    { MP_OBJ_NEW_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_GPIO_09) },
    { MP_OBJ_NEW_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_GPIO_10) },
    { MP_OBJ_NEW_QSTR(MP_QSTR_ESP_TX), MP_ROM_PTR(&pin_GPIO_09) },
    { MP_OBJ_NEW_QSTR(MP_QSTR_ESP_RX), MP_ROM_PTR(&pin_GPIO_10) },
    { MP_OBJ_NEW_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_GPIO_10) },
    { MP_OBJ_NEW_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_GPIO_09) },
```
Before my change there are a magnificent _THREE_ different pairs of UART RX/TX pins available!

Here's what I think I see: The two MP_QSTR_TX match up and the two MP_QSTR_TX match up (however they're in opposite order so you see 09/10 in one case and 10/09 in the other)

However, ESP_TX and ESP_RX are backwards -- they don't match RX and TX.  **DOUBLE HOWEVER** I checked on how the PyPortal calls its ESP_RX/TX pins  It connects the pin called ESP_TX to the ESP32 module's "TXO" pin, even though that makes it a receiving pin into CircuitPython.  That is, these are "supposed to be" backwards as they're named from the ESP32's point of view.

esp32 portion of the metro m7 rev a schematic (slightly odd appearance due to importing in kicad instead of using eagle):
image

esp32 portion of the pyportal schematic:
image

So in summary, I think the change is right, the ESP_RX/TX are either right or "right", and I'd appreciate another look from you @dhalbert

**Comment by dhalbert** [comment]
@jepler did you mean to close this, or did you do this change in another PR?

**Comment by jepler** [comment]
oops, glad you spotted my mistake

**Comment by ladyada** [comment]
`  { MP_OBJ_NEW_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_GPIO_09) }, ` is the correct RX pin

**Review by ladyada — APPROVED** [review]
